docs/installing-solidity.rst

```diff
@@ -22,7 +22,7 @@ Remix
 
 `Access Remix online <https://remix.ethereum.org/>`_, you don't need to install anything.
 If you want to use it without connection to the Internet, go to
-https://github.com/ethereum/browser-solidity/tree/gh-pages and download the .ZIP file as
+https://github.com/ethereum/remix-ide/tree/gh-pages and download the .ZIP file as
```

Comment:
The branch should be `remix_live` ( https://github.com/ethereum/remix-live/tree/gh-pages )
would it be possible to add a link to the release page?
Each release contains a little section on how to run it locally 
https://github.com/ethereum/remix-ide/releases

Reply:
Could you create a new PR with proper updates to this file?

Reply:
We used to have a version that you could just download and run in a browser - does this not exist anymore?